python/pyspark/sql/tests/test_udf.py

```diff
@@ -616,6 +616,20 @@ def f():
 
         self.spark.range(1).select(f()).collect()
 
+    def test_worker_original_stdin_closed(self):
+        # Test the original stdin of worker inherit from daemon is closed
+        # and is replaced with '/dev/null'.
+        # See SPARK-26175
+        def task(iterator):
+            import sys
+            res = sys.stdin.read()
+            # Because the stdin is replaced with '/dev/null'
+            # Read data from it will get EOF
+            assert res == '', "Expect read EOF from stdin."
```

Comment:
Shell we do `b''` instead of `''` so that we don't rely on implicit unicode / string / bytes conversion in Python 2 and 3?